Filter/ClassFilter.php

```diff
@@ -13,10 +13,11 @@
 
 use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
 use Sonata\CoreBundle\Form\Type\EqualType;
-use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
 
 class ClassFilter extends Filter
 {
+    const OPTION_SUB_CLASSES = 'sub_classes';
```

Comment:
@eugenekurasov ok, but this pollutes the global scope. If you want to avoid repeating literals, please just use a variable (technically, a variable that never changes is... a constant). Note that with php 7.1 you could make the constant private, which would be a bit better, but would still make the constant available to the rest of the class.